style-guide.md

```diff
@@ -0,0 +1,51 @@
+# WDL Style Guide
+
+All rules below should be followed by contributers to this repo. Pull Requests which do not conform to these specifications will be asked to change.
+
+## Rules
+
+- All WDL should be written in v1.0
```

Comment:
My two cents: [the history of the WDL 1.1 doc](https://github.com/openwdl/wdl/commits/main/versions/1.1/SPEC.md) suggests that the spec has been adopted for at least two years. To that end, I would not characterize it as particularly "cutting edge". This then turns into a question of "should we lag our support because Cromwell/Terra don't support the latest version?" In my mind, particularly if we are using miniwdl and miniwdl supports v1.1, I think the answer to that question should be "no".

Reply:
To be blunt, I also have no confidence in the Broad/Cromwell/Terra teams. They are already 2+ years behind while completely ignoring multiple comments, issues, etc. on both GitHub and JIRA. There was one blog post by Terra end of 2022 that indicated support in Q1 2023, but that has gone with zero update from that team.

So I agree that we should decide if we want 1.1 features and people will just have to use `miniwdl`. We also have the opinion that `miniwdl` is the better tool and a better user experience.

Reply:
My vote would also be to update to v1.1 sooner rather than later. But when we do that, we also need to update `plaid` which currently uses cromwell. Just an extra complication to consider.

I think we wait till `miniwdl-lsf` enters production (any update on that @adthrasher ?). Once that's in prod, we update this repo to 1.1

Reply:
I like that plan.